apps/site/lib/site_web/templates/trip_compare/index.html.eex

```diff
@@ -0,0 +1,49 @@
+<div class="container">
+  <h1 class="trip-plan-title">How Will the Proposed Bus Network Redesign Affect Your Trip?</h1>
+  <div class="page-section">
+    Fill in your trip details below and select the itineraries you'd like to compare.
+    You'll see how your current route and the
+    <a href="https://www.mbta.com/projects/bus-network-redesign" target="_blank">proposed redesign</a>
+    would differ in travel time, cost, accessibility and more.
+  </div>
+  <div class="page-section">
+    Then, share your thoughts with us about the changes to your trip by XX/XX/2022.
```

Comment:
Are we gonna update this with a date before merging?

Reply:
Yup yup. I'm following up on one last round of copy updates today.